eng/update-vmr.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+set -euo pipefail
+
+time="$(date +%m%d%y-%H%M)"
+
+print-help ()
+{
+   echo "Updates the VMR"
+   echo
+   echo "Syntax: ./update-vmr.sh --releaseChannel 7.0 --sdkVersion 7.0.100 --upstream <valid git url> --tarball dotnet-sdk-source.tar.gz"
+   echo
+   echo "options:"
+   echo "--releaseChannel, -r  The .NET SDK release channel (e.g. 6.0)"
+   echo "--sdkVersion, -v      The .NET SDK version (e.g. 6.0.110)"
+   echo "--upstream, -u        A valid git URL to the upstream repo to base the update branch off of"
+   echo "--tarball, -t         The tarball to update the VMR with"
+   echo "--isDryRun, -d        If set, then don't push results to AzDo"
+   echo "--help, -h            (Optional) print this help message and exit"
```

Comment:
What makes this optional and the rest required?  I'm not seeing any enforcement of the required options.  DryRun shouldn't be required but the rest should.